docs/documentation/make-first-prototype/branching.md

```diff
@@ -0,0 +1,78 @@
+# Branching (showing pages depending on user input)
+
+Our first question asks the user how many balls they can juggle. We’re going to send them to an ‘ineligible’ page if they can only juggle 2 or less.
+
+To do this, we’re going to need an ‘ineligible.html’ page, and some logic to decide when to send them there.
+
+This kind of logic goes in a file called `app/routes.js`. Routes tell the kit what to do when the user goes to specific pages, and it’s written in JavaScript.
+
+The route takes the answer the user gave to the first question and either shows them the next question or the ineligible page, depending on their answer.
+
+## Create the ineligible page
```

Comment:
```suggestion
## Create an ineligible page
```

Reply:
In the opening section above, we first refer to "an" ineligible page. Also, in the content underneath this heading, step 1 also refers to "an" ineligible page.